### ISSUE TITLE
OSM Public Strategy Meeting July 2018

### ISSUE THREAD

**Issue by mattodd** [OpenSourceMalaria/OSM_To_Do_List#568]
Hi all. High time we had another online OSM meeting. I'd really like to do this next week if people are amenable.

It'd be very good to review the latest potency and metabolic data, and plan. The plan needs to cover the next 2 months, during which I'll be largely out of action since I'm on a 2-month vacation between leaving Sydney Uni and starting in London. During that time @edwintse @maratsydney @mbhebhe @david1597 will be working as normal in Sydney (though @david1597's position comes to an end towards the end of July.

Lots to do at the moment. I'll gradually assemble an agenda below and will adapt it to any comments.

**Timing**: Sydney 6pm is EU early morning, which is quite good. Can I suggest Thursday July 5th? [Here](https://www.timeanddate.com/worldclock/meetingtime.html?iso=20180705&p1=224&p2=250&p3=136&p4=87&p5=176&p6=240)'s the timezone comparison.

*It'd be nice to have one-slide updates from anyone who wants to speak*, _BUT_: Duration: 1 hour, ideally, so preliminary discussion can always happen here.

**Platform**: I've been impressed by [Zoom](https://zoom.us/), but meetings are limited to 40 mins on their free plan. Difficult to justify paying when I can use Adobe Connect for free via Sydney Uni. Will post the meeting room link here when we've decided on a time.

**AGENDA**

1) Review of the OHOH compound (David)
2) Review of recent potency data (Ed)
3) Review of recent synthetic chemistry and telesubstitution results (Marat)
4) Review of new metabolic data (Mat)
5) Resulting priority actions for Series 4 in July and August (Mat)
6) Actions towards completion of OSM Series 4 Paper 1 (Mat)
7) Review and priority actions for Series 3 (Tha)
8) Status and next steps of pharmacophore competition Part 2 (Mat)
(Other people who want to speak to one slide - please comment below and I'll insert you - @mcoster @holeung etc etc?)
X) AOB

**Comment by drc007** [comment]
OK with me,

Chris

**Comment by MedChemProf** [comment]
It will be a 4am meeting for me, but I am fine with that. Important we meet.

**Comment by MFernflower** [comment]
6pm Sydney time is a bit rough for me but I will attempt to set reminders
on my phone as this is a super important meeting @mattodd I wish you the
best of luck on the lab move!

**Comment by holeung** [comment]
I can make it. I can provide 1-2 slides.

**Comment by david1597** [comment]
@drc007 Yes. Anyone who'd like to speak - please send your single slide to @mattodd.

**Comment by mattodd** [comment]
Draft 1 of the slide deck is linked below. I'll generate v2 in about 12 hours when I've remaining inputs and am more awake. Note everyone: the URL you'll need for the meeting, up top of the page.
@MedChemProf did you have a slide I can add?
Anyone else? Please remember we only have an hour this time, and there's too much going on to fit into that time, so we need to be brief and selective. I'd like to focus on action items for the next 2 months as the top priority.

[OSM Meeting July 2018 v1.pptx](https://github.com/OpenSourceMalaria/OSM_To_Do_List/files/2163463/OSM.Meeting.July.2018.v1.pptx)

**Comment by MedChemProf** [comment]
@mattodd I will be in attendance, but no slide input. Summer research students did not materialize, so synthesis of new targets not started. Additional manpower will be following late summer, early fall.

**Comment by mcoster** [comment]
Sorry I can't make it! School holiday duties... Here's is a very quick, rushed slide with our current efforts. Hope it goes well!

[OSM.Meeting.July.2018_COSTER.pptx](https://github.com/OpenSourceMalaria/OSM_To_Do_List/files/2164539/OSM.Meeting.July.2018_COSTER.pptx)

**Comment by mattodd** [comment]
Updated deck, now v2. Happy to take suggestions/additions right up until meeting time, in about 4 hours.

[OSM Meeting July 2018 v2.pptx](https://github.com/OpenSourceMalaria/OSM_To_Do_List/files/2164895/OSM.Meeting.July.2018.v2.pptx)

**Comment by MFernflower** [comment]
@mattodd Dont know if this is worthy for inclusion but this was my proposed S3 class compound that would more than likely just target CA - no clue if it's worth making 
![3 u2010 1 u2010benzothiophen u20102 u2010yl benzenesulfonamide](https://user-images.githubusercontent.com/3164942/42303040-ed619542-7fec-11e8-8b37-8ccf404f1b38.png)

Also included is a diamino S3 compound designed to look more ATP like'

![3- 4 2diaminothieno 3 2-d pyrimidin-6-yl benzenesulfonamide](https://user-images.githubusercontent.com/3164942/42303059-074c24f4-7fed-11e8-83d4-579ae0ad4fc6.png)

**Comment by david1597** [comment]
There was a question during the meeting about which species were used for the Pfizer biosynthesis. I forgot who asked - sorry. [Initial tests](https://github.com/OpenSourceMalaria/OSM_To_Do_List/issues/513#issue-240434518) used ten different batches from various species and four cytosols. Various species of liver microsomes gave the conversion, no conversion seen in cytosol. For the scaled-up biosynthesis that produced the samples which were sent for potency testing - it was dog liver microsomes. It was the exact same batch of dog liver microsomes used in both of the experiments that were sent for testing.

**Comment by cdsouthan** [comment]
Pending minutes I guess testing a set of front-runners (as diverse as we may have) against the  NITD609-RDd2  PfATP4 mutant and wild-type comes pretty high.  Maybe Kirk lab could do this for us? 

N.b. the PubChem IDs for the  top-9 MMVs are here   http://www.citeulike.org/user/cdsouthan/article/14611970
